docker/data-science/notebooks/ch15/0.setup.ipynb

```diff
@@ -0,0 +1,64 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ltr.download import download, extract_tgz\n",
+    "from ltr.helpers.movies import indexable_movies\n",
+    "import tarfile\n",
+    "import json\n",
+    "\n",
+    "dataset = [\"https://github.com/ai-powered-search/tmdb/raw/main/judgments.tgz\", \n",
+    "           \"https://github.com/ai-powered-search/tmdb/raw/main/movies.tgz\"]\n",
+    "download(dataset, dest=\"data/\")\n",
+    "extract_tgz(\"data/movies.tgz\") # -> Holds \"tmdb.json\", big json dict with corpus\n",
+    "extract_tgz(\"data/judgments.tgz\") # -> Holds \"ai_pow_search_judgments.txt\", \n",
+    "                                  # which is our labeled judgment list\n",
+    "\n",
+    "movies = indexable_movies(movies=\"data/tmdb.json\")\n",
+    "\n",
+    "tmdb_collection = engine.get_collection(\"tmdb\")\n",
+    "tmdb_collection.write(list(movies))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy\n",
+    "model = CLIPModel.from_pretrained(\"openai/clip-vit-base-patch32\")\n",
```

Comment:
Getting this:
`NameError: name 'CLIPModel' is not defined`

There should be a cell that looks something like this that's not here:

```
from transformers import CLIPProcessor, CLIPModel
import torch

model_id = "openai/clip-vit-base-patch32"

processor = CLIPProcessor.from_pretrained(model_id)
model = CLIPModel.from_pretrained(model_id)

# move model to device if possible
device = 'cuda' if torch.cuda.is_available() else 'cpu'

model.to(device)
```

Reply:
Stubbed code has been removed